src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import core.basesyntax.impl.BalanceHandlerImpl;
+import core.basesyntax.impl.PurchaseHandlerImpl;
+import core.basesyntax.impl.ReturnHandlerImpl;
+import core.basesyntax.impl.SupplyHandlerImpl;
+import impl.DataConverterImpl;
+import impl.DataProcessorImpl;
+import impl.ReaderFromFileImpl;
+import impl.ReportCreatorImpl;
+import impl.WriterToFileImpl;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import model.Operation;
+import service.DataConverter;
+import service.DataProcessor;
+import service.ReaderFromFile;
+import service.ReportCreator;
+import service.WriterToFile;
+import strategy.OperationStrategy;
+
+public class Main {
+    private static final String FILE_NAME = "src/main/resources/During the day.csv";
+    private static final String REPORT_FILE_NAME = "src/main/resources/Report file.csv";
```

Comment:
Avoid hardcode in your solution. It's better to pass file paths as parameters to your program, for example through command line arguments or configuration files. Hardcoded values should only be used in the Main class and/or test classes.